.circleci/config.yml

```diff
@@ -1,129 +1,231 @@
 version: 2.1
 
+parameters:
+  plugin_test:
+    type: boolean
+    default: False
+  test_plugins:
+    type: string
+    default: ""
+  cache_key_version:
+    type: string
+    default: "v0"
```

Comment:
quotes are generally optional in yaml files.

Reply:
thx, i missed this one :)